internal/pkg/manager/workloadannotator/workloadannotator.go

```diff
@@ -162,6 +197,49 @@ func (r *PodReconciler) updatePodReferences(ctx context.Context, sp *seccompprof
 	return nil
 }
 
+// updatePodReferencesForSelinux updates a SelinuxProfile with the identifiers of pods using it and ensures
+// it has a finalizer indicating it is in use to prevent it from being deleted.
+func (r *PodReconciler) updatePodReferencesForSelinux(ctx context.Context, se *selinuxprofileapi.SelinuxProfile) error {
```

Comment:
I was concerned about his as well, I will fix this

Reply:
@jhrozek 
I fixed all the other issues, I am wondering should I have another PR to implement this interface as well as the `impl` interface so I can add unit tests for `getSelinuxProfilesFromPod`? (Since we were calling the client to verify the label)

I think I need to do some learning on how to use `counterfeiter`.

Reply:
Can you file a GH issue and assign it to yourself? Then I'd personally be fine with committing a little code duplication as long as we work on fixing it.

Reply:
Thanks! I will create an issue for it.